src/monaco/browser/monaco-command.ts

```diff
@@ -48,6 +48,17 @@ export class MonacoEditorCommandHandlers implements CommandContribution {
             commands.registerHandler(id, handler);
         });
 
+        [CommonCommands.SWITCH_HEADER_SOURCE].forEach(id => {
```

Comment:
the same here, should not be here, the cpp extension can use the monaco extenion but not another way around